.github/build-cop-template.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: Build Cop Log {{ date | date('ddd, D MMM YYYY') }}
+title: Build Cop Log - {{ date | date('ddd, D MMM YYYY') }}
 labels: build-cop-rotation
 ---
 
-1. Login to https://builds.spinnaker.io and ensure all builds are green
-2. If failing, comment on this issue any actions you had to perform to get it back to green.
-3. Close this issue when finished.
+1. Login to [https://builds.spinnaker.io](builds.spinnaker.io).
```

Comment:
This is backwards... it should be `[builds.spinnaker.io](https://builds.spinnaker.io)`.

Reply:
Good catch.